### ISSUE TITLE
UIControl slider inside content view

### ISSUE THREAD

**Issue by mnizar** [SCENEE/FloatingPanel#125]
Hi, 

in contentViewController i have a view slider (TTRangeSlider) and it's not working when I try to move slider, the floating panel always try to dismissed, but the slider working when i set `filterPanelVC.panGestureRecognizer.isEnabled = false`

I've tried add this in delegate method but still not working :
```
func floatingPanel(_ vc: FloatingPanelController, shouldRecognizeSimultaneouslyWith gestureRecognizer: UIGestureRecognizer) -> Bool {
        switch vc {
        case filterPanelVC:
            return (vc.contentViewController as? CategoryFilterVC)?.priceSlider.gestureRecognizers?.contains(gestureRecognizer) ?? false
        default:
            return false
        }
    }
```

Any idea to make both swipe to dismiss and the slider working ?

Thank you

**Comment by SCENEE** [comment]
Thanks for your report, @mnizar! TTRangeSlider handles touches directly without any gesture recognizers. Therefore, could you please try this setting?

```swift
filterPanelVC.panGestureRecognizer.cancelsTouchesInView = false
```

If it works, I'm going to modify the library to support the settings by default.

**Comment by mnizar** [comment]
Hi, 

it's work with set `panGestureRecognizer.cancelsTouchesInView = false`
but the panel always bouncing when i try to change slider, and sometimes the panel is closed
here's attached example project

Thank you
[FloatingPanelSlider.zip](https://github.com/SCENEE/FloatingPanel/files/2832189/FloatingPanelSlider.zip)

**Comment by SCENEE** [comment]
I'm sorry to be late for my reply. Thanks for your example code!
This issue will be resolved by using  `floatingPanelShouldBeginDragging(_:)` delegate method to control moving a panel. It's developing on `improve-operability` branch. I will be merged when I release on v1.4.0.

**Comment by SCENEE** [comment]
Ref #193
